modules/rostests/apitests/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ add_subdirectory(gdi32)
 add_subdirectory(gditools)
 add_subdirectory(imm32)
 add_subdirectory(iphlpapi)
+if(ISAPNP_ENABLE)
```

Comment:
Why do we have a build define for this, by the way?

Reply:
This is an optimization for the build process so it won't need to build the entire driver for a non-x86 architecture. Note that the driver busy loops on a QBR handler until it reaches the last read data port in the detection list.

Reply:
> [...] so it won't need to build the entire driver for a non-x86 architecture.

Isn't it possible to use a `ARCH STREQUAL ...` or something similar then?

Reply:
https://github.com/reactos/reactos/pull/3467#discussion_r650448983
Victor asked me to move the `ARCH STREQUAL ...` line into the cmake configuration file. I think it's helpful to keep everything related to the same ARCH in one place.

Reply:
I don't know, he didn't give an explanation why using ISAPNP_ENABLE instead of the ARCH thing.